demos/JWST/S1_miri_template.ecf

```diff
@@ -26,6 +26,16 @@ skip_gain_scale     False
 #Pipeline stages parameters
 jump_rejection_threshold  4.0 #float, default is 4.0, CR sigma rejection threshold
 
+#Background subtraction
+grouplevel_bg 		True
```

Comment:
Please don't use tabs in the ECF/EPF templates as it messes up the formatting for people depending on what file viewer they're using